docs/examples/llm/llama_2_llama_cpp.ipynb

```diff
@@ -55,7 +55,7 @@
    "source": [
     "from llama_index.core import SimpleDirectoryReader, VectorStoreIndex\n",
     "from llama_index.llms.llama_cpp import LlamaCPP\n",
-    "from llama_index.core.llms.llama_utils import (\n",
+    "from llama_index.llms.llama_cpp.llama_utils import (\n",
```

Comment:
don't think this is the correct path for 0.10.*

Reply:
I'm using version 10.4 and it works. Can you test it?